src/spec/PointerInputSpec.ts

```diff
@@ -0,0 +1,101 @@
+/// <reference path="jasmine.d.ts" />
+
+/// <reference path="Mocks.ts" />
+
+describe('A pointer', () => {
+   
+   var mockWindow = null;
+   var engine: ex.Engine = null;
+   var pointers: ex.Input.Pointers = null;
+   var mocker = new Mocks.Mocker();
+   var noop = () => { return null; };
+
+   beforeEach(() => {
+      mockWindow = <any>mocker.window();
+      engine = mocker.engine(600, 400);
```

Comment:
I think we want to try switching over to `TestUtils.engine` instead for new tests, which is a real engine instance that can be tested. You will need:

```
afterEach(() => {
  engine.stop();
});
```

As well, to make sure it gets stopped after each test.